drivers/note/Kconfig

```
@@ -87,6 +87,24 @@ config DRIVERS_NOTERAM_CRASH_DUMP
 
 endif # DRIVERS_NOTERAM
 
+config DRIVERS_NOTELOWEROUT
+	bool "Note lower output"
+	default n
+	---help---
+		Note uses lower output.
+
+config DRIVERS_NOTEFILE
+	bool "Note file driver"
+	---help---
+		The note driver output to file.
```

Since Note is the name of this board, would it be helpful to capitalize it (as Note, rather than note) everywhere for consistency? 

```suggestion
		The Note driver output to file.
```

---

Done